jaba/src/pages/LoginWelcomePage/LoginWelcomePage.module.css

```diff
@@ -0,0 +1,115 @@
+
+
+.splitScreen {
+    height: calc(100vh);
+    display: flex;
+    flex-direction: row;   
+}
+
+.leftSide {    
+    width: 50%;    
+    display: flex;    
+    align-items: center;
+    justify-content: center;
+    background-color: #023C47;
+}
+
+.welcomeRectangle {
+    width: 505px;
+    height: 551px;
+    top: 216px;
+    left: 125px;
+    border-radius: 10px;
+    border: 0.5px;
+    background: linear-gradient(0deg, #FFFFFF, #FFFFFF);
+    border: 0.5px solid #878787
+}
+
+.rightSide {    
+    width: 50%;    
+    display: flex;    
+    align-items: center;
+    justify-content: center;
+}
+
+.rightImage {
+    width: 45%;
+    height: 45%;
+}
+
+.alignWelcome {
+    text-align: left;
+    padding-top: 80px;
+    padding-left: 43px;
+    font-family: 'DM Serif Text', serif;
+    font-size: 31px;
+}
+
+.alignWelcomeBlurb {
+    text-align: left;
+    font-family: 'Inter', serif;
+    padding-left: 43px;
+    padding-right: 43px;
+    font-size: 16px;
+
+}
+
+.alignWelcome, .alignWelcomeBlurb {
+    margin-bottom: -25px; /* Adjust this value as needed to control the spacing */
+  }
+
+  .alignWelcomeBlurb, .signInUserBox {
+    margin-bottom: -35px; /* Adjust this value as needed to control the spacing */
+  }
+
+.signInUserBox{
+    padding-left: 43px;
+    padding-right: 43px;
+    padding-top: 100px;
+    margin-bottom: -81px;
+}
+
+.signInAdminBox{
+    padding-left: 43px;
+    padding-right: 43px;
+    padding-top: 100px;
+    margin-bottom: 35px;
+}
+.signInUserButton {
```

Comment:
Can you style some properties of .signInUserButton and .signInAdminButton at once? They share a good number of CSS properties

Reply:
i feel like it's better to have them seperate? just in case we want to change one in the future, but also because they're different colors so they'd hve to have different catagories regardless

Reply:
I'm going to still style some properties at once to reduce CSS code. Your point is still valid but I think it would be better to jut merge some styling to reduce redundancy since its just styling on two buttons and if the style on one changes, it sholdn't be too difficult to adapt still